specs/scenarios/trust-store-and-trust-policy.md

```diff
@@ -0,0 +1,212 @@
+            ]
+        }
+    ]
+}
+```
+
+### Adding Trust Certificates to the Trust Store and Trust Policy for the Application Layer
+Once the node is started and the Wabbit Networks Batch service is deployed on the node, a customer can deploy their workloads. For that to work, the *trust store* needs to be updated with the Wabbit Networks Batch certificates and a trust policy needs to be added. Here is how the *trust store* should look like:
+
+```
+$XDG_CONFIG_HOME/notation/trust-store
+    /x509
+        /ca
+            /wabbit-networks
+                wabbit-networks-kubernetes.crt
+                wabbit-networks-batch.crt
+            /acme-rockets
+                acme-rockets-mr.crt
+        /tsa
+            /publicly-trusted-tsa
+                tsa-cert1.pem
+```
+And here is how the policy should look like before deployint the ACME Rockets Map/Reduce application:
+
+```
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "wabbit-networks-kubernetes",
+            "registryScopes": [ "k8s.wabbitnetworkscr.io/k8s" ],          
+            "signatureVerification": {          
+              "level" : "strict" 
+            },
+            "trustStores": ["ca:wabbit-networks"], 
+            "trustedIdentities": [
+              "x509.subject: C=US, ST=WA, L=Seattle, O=wabbitnetworks.com, OU=KubernetesTeam, CN=KuberenetesTeamSecureBuilder"
+            ]
+        },
+        {
+            "name": "wabbit-networks-batch",
+            "registryScopes": [ "batch.wabbitnetworkscr.io/batch" ],          
+            "signatureVerification": {          
+              "level" : "strict" 
+            },
+            "trustStores": ["ca:wabbit-networks"], 
+            "trustedIdentities": [
+              "x509.subject: C=US, ST=WA, L=Seattle, O=wabbitnetworks.com, OU=BatchTeam, CN=BatchTeamSecureBuilder"
+            ]
+        },
+        {
+            "name": "acme-rockets-mapreduce",
+            "registryScopes": [ "mr.acmecr.io/mr" ],          
+            "signatureVerification": {          
+              "level" : "strict" 
+            },
+            "trustStores": ["ca:acme-rockets"], 
+            "trustedIdentities": [
+              "x509.subject: C=US, ST=WA, L=Seattle, O=acmerockets.com, OU=MapReduceTeam, CN=MapReduceTeamSecureBuilder"
+            ]
+        }
+    ]
+}
+```
+
+#### Removing Trust Certificates from the Trust Store and the Trust Policy
+When the customer's workload is shut down, the trust established for the customer's certificate should be removed from the node. In cases as above, the customer workloads may be temporary (for example while running the map/reduce task) and once they are done, the trust to the customer certificate should be removed from the nodes. There shouldn't be expectation that a second batch from the same customer will be scheduled on the same cluster/nodes.
```

Comment:
who will delete the trust store and trust policy?

Reply:
Same answer as above. Any tool should be able to do that and we should enable it through APIs and CLI commands in Notation.